deps.windows/90-vst3.ps1

```diff
@@ -0,0 +1,61 @@
+param(
+    [string] $Name = 'vst3sdk',
+    [string] $Version = 'v3.8.0',
+    [string] $Uri = 'https://github.com/steinbergmedia/vst3sdk.git',
+    [string] $Hash = '9fad9770f2ae8542ab1a548a68c1ad1ac690abe0',
+    [array] $Targets = @('x64', 'arm64')
+)
+
+function Setup {
+    Log-Information "Setup (${Target})"
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
```

Comment:
It _may_ be faster to manually call `Invoke-GitCheckout` with `-Sparse`:
https://github.com/obsproject/obs-deps/blob/2a08bc9f5996efb0627a66dacdb74727d8c79459/deps.ffmpeg/80-amf.ps1#L14

Otherwise, we'd have to update `Setup-Dependency` to accept the `-Sparse` parameter and arguments.

Reply:
I initially used a sparse checkout, see the exchange with Pat. But he advised to use Setup-Dependency, arguing that speed doesn't matter. See: https://github.com/obsproject/obs-deps/pull/303#discussion_r2494096494 and following exchange